pkg/machines/components/create-vm-dialog/password.css

```diff
@@ -0,0 +1,23 @@
+/* user config - password strength */
+.password-strength-meter {
+    height: 5px;
+    margin-top: 5px;
+    margin-bottom: 0;
+}
+.password-strength-meter div {
+    width: 25%;
+    border-right: 3px solid #fff;
+    background-color: transparent;
+}
+.password-strength-meter.weak div:first-child {
+    background-color: var(--color-red-1);
+}
+.password-strength-meter.okay div:nth-child(-n+2) {
+    background-color: #EC7A08;
+}
+.password-strength-meter.good div:nth-child(-n+3) {
+    background-color: #3F9C35;
+}
+.password-strength-meter.excellent div:nth-child(-n+4) {
+    background-color: #3F9C35;
+}
```

Comment:
This just cries for a PF4 standard component :-)

Reply:
There is not one yet unfortunately - this was shamelessly stolen frmo composer code.